setup.cfg

```diff
@@ -38,7 +38,7 @@ exclude =
     __pycache__/,
     node_modules/,
     */migrations/
-ignore = H101,H238,H301,H306,W503
+ignore = E203,H101,H238,H301,H306,W503
```

Comment:
It is because black and flake8 conflict with this check. Example:

```python
# black
value = fruits[start : length - 1]

# flake8
value = fruits[start:length - 1]
```

Reply:
Maybe we should turn off an error by line? 
(I don't know in how many lines it appears.)

Reply:
It's honestly rate. When I want to avoid the conflict I would just do:
```python
a = x + 2
b = y - 1
value = fruits[a:b]
```
Then it won't conflict.